libs/sdk-ui-ext/src/internal/utils/internalIntlProvider.tsx

```diff
@@ -14,7 +14,7 @@ import ptPT from "../translations/pt-PT.json";
 import zhHans from "../translations/zh-Hans.json";
 import { DefaultLocale, ILocale } from "@gooddata/sdk-ui";
 
-const messagesMap = {
+export const messagesMap = {
```

Comment:
small stuff: the messagesMap in sdk-ui has this kind of typing:
`export const messagesMap: { [locale: string]: ITranslations }`

Reply:
It is not critical, but it could help us, I'll add a similar type (the ITranslations type is internal to sdk-ui and I think it should not be exported).